packages/components/psammead-brand/src/index.jsx

```diff
@@ -45,10 +45,10 @@ const StyledWrapper = styled.div`
 const ConstraintWrapper = styled.div`
   max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
   margin: 0 auto;
+  padding-top: ${GEL_SPACING};
```

Comment:
Does moving this padding top from the `<a>` to here not reduce the click area? UX had explicitly set the click area to be as big as possible EG: the height of the banner

Reply:
It does reduce the clickable area. Nice catch. Fixed it.